lib/ZwaveClient.js

```diff
@@ -899,13 +922,14 @@ function initNode (zwaveNode) {
   node.productType = zwaveNode.productType
   node.manufacturer = deviceConfig.manufacturer
   node.firmwareVersion = zwaveNode.firmwareVersion
-  node.zwaveVersion = zwaveNode.version
+  node.protocolVersion = zwaveNode.protocolVersion
   node.zwavePlusVersion = zwaveNode.zwavePlusVersion
-  node.nodeType = zwaveNode.nodeType
-  node.roleType = zwaveNode.roleType
+  node.nodeType = zwaveNode.zwavePlusNodeType
+  node.roleType = zwaveNode.zwavePlusRoleType
```

Comment:
Are you not interested in the new `nodeType` (controller / slave) property?

Reply:
And since you're dumping all of this info, don't you want to know or display the (max) data rate(s)?

Reply:
Forgot to dump that, adding it now